app/views/govuk_component/breadcrumbs.raw.html.erb

```diff
@@ -1,18 +1,21 @@
 <%
   breadcrumbs ||= []
+  collapse_on_mobile ||= false
 %>
-<div class="govuk-breadcrumbs" data-module="track-click">
+
+<div class="govuk-breadcrumbs <%= "collapse-on-mobile" if collapse_on_mobile %>" data-module="track-click">
   <ol>
   <% breadcrumbs.each_with_index do |crumb, index| %>
-    <li>
-      <%
-        is_link = crumb[:url].present? || crumb[:is_current_page]
-        path = crumb[:is_current_page] ? '#content' : crumb[:url]
-        aria_current = crumb[:is_current_page] ? 'page' : 'false'
-        css_class = crumb[:is_current_page] ? 'breadcrumb-for-current-page' : ''
+    <%
+      is_link = crumb[:url].present? || crumb[:is_current_page]
+      is_parent = index == breadcrumbs.length - 2
```

Comment:
This won't work on any breadcrumbs which don't have the current page breadcrumb (which is currently optional).